src/General/@next/Button/ButtonStyle.ts

```diff
@@ -0,0 +1,118 @@
+import { CSSProperties } from 'react';
+import styled from 'styled-components';
+import { Breakpoints } from '../../..';
+import { borderRadius4 } from '../../../Utils/@next/borderRadius';
+import { Neutral } from '../../../Utils/@next/colors';
+import {
+  button,
+  buttonFocused,
+  insetButtonPressed,
+} from '../../../Utils/@next/dropShadow';
+import {
+  space12,
+  space16,
+  space24,
+  space4,
+  space8,
+} from '../../../Utils/@next/spacing';
+import { ButtonProps, ButtonSize } from './Button';
+
+const defaultButtonSizeStyle = `
+  width: 75px;
+  height: 36px;
+  padding: ${space8} ${space16};
+
+  @media (max-width: ${Breakpoints.large}) {
+    width: 71px;
+  }
+`;
+
+const largeButtonSizeStyle = `
+  width: 91px;
+  height: 44px;
+  padding: ${space12} ${space24};
+
+  @media (max-width: ${Breakpoints.large}) {
+    width: 87px;
+  }
+`;
+
+const slimButtonSizeStyle = `
+  width: 67px;
+  height: 32px;
+  padding: ${space4} ${space12};
+
+  @media (max-width: ${Breakpoints.large}) {
+    width: 63px;
+  }
+`;
+
+const buttonSizeStyleMap: Record<ButtonSize, string> = {
+  default: defaultButtonSizeStyle,
+  slim: slimButtonSizeStyle,
+  large: largeButtonSizeStyle,
+};
+
+export const BasicButton = styled.button.attrs(
+  ({ active, fullWidth }: ButtonProps) => {
+    const activeStyle = active
```

Comment:
you can remove the ternary operators and return style as:

```ts
{
  style: {
    ...(active && activeStyle),
    ...(fullWidth && widthStyle)
  }
}
```

Reply:
That looks better, thanks!